docs/html/conf.py

```diff
@@ -275,9 +275,13 @@
 
 # Here, we crawl the entire man/commands/ directory and list every file with
 # appropriate name and details
-for fname in glob.glob('man/commands/*.rst'):
-    fname_base = fname[4:-4]
-    outname = 'pip-' + fname_base[13:]
+man_dir = os.path.join(docs_dir, 'man/')
+raw_subcommands = glob.glob(os.path.join(man_dir, 'commands/*.rst'))
+if len(raw_subcommands) < 1:
```

Comment:
Just an `assert raw_subcommands, "Did not find subcommand man pages"` is clearer here. 🙃

Reply:
I sort of feel instinctively afraid to use assertions outside of an e.g. unittest environment. `sphinx-build` is a CLI command, how should I know if it will be optimized away to nothing?

Admittedly if it is wrong once, it is probably wrong everywhere, but the point still stands: I don't gravitate towards using them....

Would you like me to change it?

Reply:
It usually isn't and it's really just a sanity check. None the less, if you prefer to not use assert here, please do change the if to be `if not raw_subcommands`.

The use of len, was what flagged my attention.

Reply:
Good point, fixed the use of len.